v1/OpsGenieConfig.dhall

```diff
@@ -0,0 +1,58 @@
+let imports = ../imports.dhall
+
+let Kubernetes = imports.Kubernetes
+
+let BasicAuth = ./BasicAuth.dhall
+
+let HTTPConfig = ./HTTPConfig.dhall
+
+let OpsGenieConfigDetail =
+      { Type = { key : Text, value : Text }, default = {=} }
```

Comment:
Just to keep the convention in the repo seperate it to different file OpsGenieConfigDetail.dhall